Xamarin.Forms.Core/Shell/ShellSection.cs

```diff
@@ -475,24 +475,6 @@ void RemoveExcessPathsWithinTheRoute()
 			}
 		}
 
-		List<Page> BuildFlattenedNavigationStack(List<Page> startingList, IReadOnlyList<Page> modalStack)
```

Comment:
This code just got moved to `ShellNavigationManager.BuildFlattenedNavigationStack`